lib/formtools/plugins/plugins-helpers.js

```diff
@@ -226,6 +226,41 @@ var overviewDefaults = function (overview, labels) {
 
     overview.summary.label = overview.summary.label || 'Summary';
 
+
+    //overview Details
+    if (!overview.details || typeof overview.details !== 'object') {
+        // defaults to {}
+        overview.details = {};
+    }
+
+    overview.details.form = overviewDetailsDefault(overview.details.fields, schema, form, labels);
+
+    //set tabs to tabsArray property of overview.details
+    var tabsArray = [];
+    if (Array.isArray(overview.details.tabs) && overview.details.tabs.length) {
+
+        for (var i = 0; i < overview.details.tabs.length; i++) {
+
+            var tempTab = {};
+            tempTab.label = overview.details.tabs[i].label;
+
+            if (Array.isArray(overview.details.tabs[i].fields)) {
+
+                tempTab.form = overviewDetailsDefault(overview.details.tabs[i].fields, schema, form, labels);
+
+            } else if (typeof overview.details.tabs[i].body === 'function') {
+                tempTab.body = overview.details.tabs[i].body;
+            }
+
+            tabsArray.push(tempTab);
+        }
+    }
+
+    //set the tabsArray to overview.details.tabsArray
+    overview.details.tabsArray = tabsArray;
```

Comment:
Could you please rename `overview.details.tabsArray` to be `overview.details.tabs`. This will keep it more consistent with the rest of Linz. We haven't been using data type based naming anywhere else.